sdk/keyvault/Azure.Security.KeyVault.Administration/src/Generated/KeyVaultAdministrationClientOptions.cs

```diff
@@ -12,6 +12,6 @@ namespace Azure.Security.KeyVault.Administration
     /// <summary> Client options for <see cref="KeyVaultAccessControlClient"/>. </summary>
     public partial class KeyVaultAdministrationClientOptions : ClientOptions
     {
-        private const ServiceVersion LatestVersion = ServiceVersion.V7_6;
+        private const ServiceVersion LatestVersion = ServiceVersion.V2025_07_01;
```

Comment:
[This CI error](https://dev.azure.com/azure-sdk/public/_build/results?buildId=5397679&view=logs&j=c3e74943-23a9-5e82-9b42-8286fce989d4&t=e049197e-d864-58b1-c0e9-38458fd26878&l=757) is because this file was modified manually. Don't do any manual edits to generated code.